packages/dx-react-scheduler/src/plugins/all-day-panel.tsx

```diff
@@ -114,17 +122,16 @@ export class AllDayPanel extends React.PureComponent {
                   endViewDate,
                   excludedDays,
                   viewCellsData,
-                  cellElements: tableRef.querySelectorAll('th'),
+                  cellElements: (tableRef! as HTMLElement).querySelectorAll('th'),
```

Comment:
This castings can be extracted to a variable: 
`const tableElement = tableRef! as HTMLElement`